### ISSUE TITLE
Consider adding an infobox to data explorer for grouped results

### ISSUE THREAD

**Issue by coffeemug** [rethinkdb/rethinkdb#2054]
If the user doesn't call `ungroup`, simply showing the results may be confusing. One option is to add an infobox for these cases.

I'm adding this to 1.12-polish. Let me play with the GMR implementation when @mlucy finishes his latest round of changes. I'll comment on this issue then to say if it's worth doing.

This shouldn't take more than a day to do, so @neumino should have plenty of time to finish it for 1.12 if we decide to get this in.

TL;DR: do nothing for now, I'll comment on this shortly.
